lib/absinthe/phase/document/arguments/parse.ex

```diff
@@ -40,7 +36,14 @@ defmodule Absinthe.Phase.Document.Arguments.Parse do
   end
 
   defp build_value(%{__struct__: struct} = normalized, %Type.Scalar{} = schema_node, context)
-       when struct in [Input.Boolean, Input.Float, Input.Integer, Input.String, Input.Null] do
+       when struct in [
+              Input.Boolean,
+              Input.Float,
+              Input.Integer,
+              Input.String,
+              Input.Null,
+              Input.Object
```

Comment:
call parse for `Input.Object` to allow consumers of absinthe to customize parsing object scalars if they want to